spec/factory_helpers/hmrc_response/use_case_one.rb

```diff
@@ -42,13 +43,147 @@ def response
         {
           'submission' => @correlation_id,
           'status' => status,
-          'data' => data_array
+          'data' => @options[:data_array] || default_data_array
         }
       end
 
       private
 
-      def data_array
+      def generate_named_data
+        case @options[:named_data]
+        when :eg1_uc1
+          eg1_uc1
```

Comment:
Naming things is hard... does eg1 mean example1?  
I wasn't sure if there was another, hidden, meaning

Reply:
Yes, - I'll rename it to `example1_usecase1`